docs/internships/outreachy/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+id: outreachy-introduction
+title: Introduction
+---
+
+Outreachy is a paid, remote internship program. Outreachy's goal is to support people from groups underrepresented in tech. We help newcomers to free software and open source make their first contributions.
+Outreachy provides internships to work open source. People apply from all around the world. Interns work remotely, and are not required to move. [Outreachy internship](https://www.outreachy.org/) projects may include programming, user experience or documentation.
+
+The outreachy internship application has two phases, the first stage is called [initial application](https://www.outreachy.org/docs/applicant/#initial-application) and the second stage is called the [contribution period](https://www.outreachy.org/docs/applicant/#application-prep). 
+
+### Eligibility Criteria
+
+Inorder to be part of outreachy internship,
```

Comment:
Please add  a link to the criteria. That way we won't have to maintain it if they change. Reduce the amount of text to match. Add the link to the section below.